tests/unit/test_charm.py

```diff
@@ -40,7 +41,7 @@ def test_misconfigured_charm_reaches_blocked_status():
     """
     harness = Harness(SamlIntegratorOperatorCharm)
     harness.begin()
-    harness.model.unit.status == ops.BlockedStatus()
+    assert harness.model.unit.status.name == ops.BlockedStatus().name
```

Comment:
Do we need an instance or is the name available directly on the class?

Reply:
The name is available

Reply:
The established pattern in most of our tests is to compare it to `ops.BlockedStatus.name`

Reply:
Noted